smartBugs.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python3
 
 import argparse
+from collections import namedtuple
```

Comment:
The best practice's import order suggests to group them as follows: first a group with the imports (alphabetically ordered) and then a group with 'from ...' (also alphabetically ordered): https://www.python.org/dev/peps/pep-0008/

So, move this collection to the import block starting in L17.